Activities/Cryptography/UiPath.Cryptography.Activities/Models/CryptographyLocalItem.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using UiPath.Cryptography.Activities.Extensions;
+using UiPath.Cryptography.Activities.Properties;
+using UiPath.Platform.ResourceHandling;
+
+namespace UiPath.Cryptography.Activities.Models
+{
+    public class CryptographyRemoteItem : IRemoteResource
```

Comment:
This should never actually be used, since crypto activities have no concept of "remote" files.